### PR TITLE
Feature/bank account class

### PR DESCRIPTION
**Pull request by charleysan**
Jess, Noah and Charles Bank Management System

### CODE REVIEW

File: spec/bankaccount_rspec.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
+require_relative '../bank_git'
+
+RSpec.describe BankAccount do
+  let(:ls_account) {BankAccount.new("Leon")}
+
+  describe '#initialize' do
+    it 'initializes the account owner and balance' do
+      expect(ls_account.owner).to eq("Leon")
```
Comment (AP-Lett) [inline spec/bankaccount_rspec.rb:8]: add test for initializing initial zero balance

**Review by AP-Lett — COMMENTED** [review]
kinda started getting lost in the sauce because of variable names. Good job, with everything though!!

File: spec/bankaccount_rspec.rb (hunk 1 above)
Comment (AP-Lett) [reply spec/bankaccount_rspec.rb:8]: You didnt have to run this, was just requesting in case you wanted to have that initialized value in your RSpec!

**Review by AP-Lett — APPROVED** [review]
<3 x 3
